widgetssdk/src/test/java/com/glia/widgets/callvisualizer/ActivityWatcherForCallVisualizerControllerTest.kt

```diff
@@ -38,8 +39,10 @@ import org.mockito.kotlin.times
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.verifyNoMoreInteractions
 import org.mockito.kotlin.whenever
+import org.robolectric.RobolectricTestRunner
 import java.util.function.Consumer
 
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Why this addition? Do we need to start adding this line to other tests?

Reply:
We need Robolectric if the code we test calls something from the `android.` package. Robolectric provides the default implementation of Android API.